techniques/css/C41.html

```diff
@@ -151,29 +151,36 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-			<h2>Related Techniques</h2>
+			<h2>関連テクニック</h2>
 			<ul>
 				<li><a href="../general/G195">G195: Using an author-supplied, visible focus indicator</a></li>
 			</ul>
 		</section>
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
-				<p>For each focusable user interface component:</p>
+				<h3>手順</h3>
+				<p>それぞれのフォーカス可能なユーザインターフェース コンポーネントに対して:</p>
 				<ol>
-                    <li>Place keyboard focus on each focusable user interface element on the page using the keyboard.</li>
-                    <li>Check that the focus indicator area is at least the size of a 2 CSS px border around the component.</li>
-                    <li>Check that the change of contrast of the indicator between focused and unfocused states has a ratio of 4.5:1 for the minimum focus indicator area.</li>
+                    <li>キーボードを使用して、ページ上のフォーカス可能な各ユーザインタフェース要素にキーボードフォーカスを当てる。</li>
+                    <li>フォーカスインジケータ領域が、コンポーネントの周囲の 2 CSS ピクセル以上のボーダーであることをチェックする。</li>
+                    <li>最小のフォーカスインジケータ領域に対して、フォーカス状態と非フォーカス状態との間のインジケータのコントラストの変化が 4.5:1 の比率であることをチェックする。</li>
 				</ol>
 			</section>
 			<section class="test-results" id="expected-results">
-				<h3>Expected Results</h3>
+				<h3>期待される結果</h3>
 				<ul>
-					<li>#2 and #3 are true.</li>
+					<li>2 及び 3 が真である。</li>
```

Comment:
ここの表現ですが、既存の他のテクニック (WCAG 2.1 達成方法集) では「#1 及び #2 の結果が真である。」のような表現になっています。

```suggestion
					<li>#2 及び #3 の結果が真である。</li>
```

Reply:
日本語では\#1や\#2というような表記をあまりしないと思うんですよね…単にこうするのがよいかと思いますがどうでしょうか？
```suggestion
					<li>2 及び 3 の結果が真である。</li>
```